vector/src/main/java/im/vector/app/features/home/room/detail/timeline/item/MessageInformationData.kt

```diff
@@ -55,6 +54,16 @@ data class ReferencesInfoData(
         val verificationStatus: VerificationState
 ) : Parcelable
 
+@Parcelize
+data class ReactionsSummaryData(
+        /*List of reactions (emoji,count,isSelected)*/
+        val reactions: List<ReactionInfoData>? = null,
+        val showAll: Boolean = false,
+        val onShowMoreClicked: () -> Unit,
+        val onShowLessClicked: () -> Unit,
+        val onAddMoreClicked: () -> Unit
```

Comment:
Weird to see some lambda in data class

Reply:
not sure what will happen when we attempt to parcelize :thinking: